src/bin/wasmer.rs

```diff
@@ -316,7 +316,9 @@ fn execute_wasm(options: &Run) -> Result<(), String> {
     };
 
     if !utils::is_wasm_binary(&wasm_binary) {
-        wasm_binary = wabt::wat2wasm(wasm_binary)
+        let mut features = wabt::Features::new();
+        features.enable_simd();
```

Comment:
I think it will better to only enable SIM when the indicated backend is LLVM
Thoughts?

Reply:
```suggestion
        if options.backend == Backend::LLVM {
            features.enable_simd();
        }
```

Reply:
Or follow the trend set by other implementations and require "--enable-simd" because it's a pre-standard feature?

Reply:
It would be nice to standardize on an env var for this 🤔 

Maybe worth posting this to the https://github.com/webassembly/design repo

Reply:
> Or follow the trend set by other implementations and require "--enable-simd" because it's a pre-standard feature?

Having it behind a flag in Google Chrome makes sense because the implications of SIMD and because of their immense usage (so they need to be 1000% sure of everything working before enabling it).

However in our case I don't see any reason for not enabling it by default for LLVM (since our usage is still small and we can afford to iterate on it while shipping something that works for most of the cases).

Reply:
I missed this conversation but posted it as a question on: https://github.com/WebAssembly/simd/issues/87